setup/install/halyard.md

```diff
@@ -21,17 +21,79 @@ just going to tell you to use Halyard.
 
 There are two ways you can install Halyard:
 
+* [locally on Debian/Ubuntu or macOS](#install-on-debianubuntu-and-macos)
 * [on Docker](#install-halyard-on-docker)
 
-  Docker installation is strongly recommended.
+## Install on Debian/Ubuntu and macOS
 
-* [on Ubuntu](#install-on-ubuntu-14041604)
+Halyard runs on...
 
-  We support installation on 14.04 and 16.04. Installing on 16.04 requires
-  Spinnaker 1.6.0 or later.
+* Ubuntu 14.04 or 16.04 (Ubuntu 16.04 requires Spinnaker 1.6.0 or later)
+* Debian 8 or 9
+* macOS
```

Comment:
What version of macOS has this been validated on? We should indicate, even if loosely. At least make clear it's not macos9 :)

Reply:
@lwander ?

Reply:
High sierra